docs/app/pages/components/components-sections/input-controls/radio-button/radio-button.component.html

```diff
@@ -98,10 +101,36 @@
 </uxd-api-properties>
 
 <p>
-    In addition to the options listed above, a radio button can be disabled using the <code>disabled</code> attribute. You can also provide
-    the radio button with a <code>name</code> or <code>id</code> if required.
+    In addition to the options listed above, a radio button can be disabled using the
+    <code>disabled</code> attribute. You can also provide the radio button with a
+    <code>name</code> or <code>id</code> if required.
+</p>
+
+<h4>Keyboard Accessibility</h4>
+<p>
+    Due to an issue within native HTML it is impossible to focus any of the
+    <code>ux-radio-buttons</code> while one is <code>selected</code> and <code>disabled</code> when
+    using the keyboard. As a workaround we recommend removing the <code>selected</code> state from
+    the <code>ux-radio-button</code> when disabling it.
 </p>
+<p>Below is a code example:</p>
```

Comment:
Sorry for the misunderstanding, if you have a workaround could you integrate it into the working example in our docs? I don't want any of our examples to be keyboard inaccessible.